Automattic/pocket-casts-ios
# Convert strings file to UTF-8

Our `.strings` files are being treated as binary files by git. This PR changes them from `UTF-16 LE` to `UTF-8`.

You won't be able to see it here on the pull request, however, if you change `podcasts/en/Localizable.strings` you'll see that git should start showing the diff.

## To test

1. Change `podcasts/en/Localizable.strings`
2. Check that is shown as a diff when you check the `git diff`

## Checklist
